functionary/prompt.py

```diff
@@ -5,6 +5,10 @@
 
 import torch
 
+from functionary.schema import generate_schema_from_functions
+
+SYSTEM_MESSAGE = """A chat between a curious user and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the user's questions. The assistant calls functions with appropriate input when necessary"""
```

Comment:
There was already a system message in the inference. Either let's delete this and import that one. Or delete the other one.